lib/setup_sharify.js

```diff
@@ -62,6 +62,7 @@ sharify.data = _.extend(_.pick(config,
   'MARKETING_SIGNUP_MODALS',
   'MAX_POLLS_FOR_MAX_BIDS',
   'METAPHYSICS_ENDPOINT',
+  'MOBILE_MARKETING_SIGNUP_MODALS',
```

Comment:
I changed name of the constant so that it didn't conflict with the desktop version.